systems/plants/constraint/RigidBodyConstraint.cpp

```diff
@@ -27,13 +27,14 @@ void drakePrintMatrix(const MatrixXd &mat)
   }
 };
 
-static void checkBodyInd(int body, int num_bodies, int min_body_ind = 0)
-{
-  if(body< min_body_ind || body>=num_bodies)
-  {
-    std::cerr<<"body should be within ["<<min_body_ind<<" robot.num_bodies-1]"<<std::endl;
-  }
-}
+//static void checkBodyInd(int body, int num_bodies, int min_body_ind = 0)
```

Comment:
fine by me, it wasn't being used but I thought maybe you might use it in the future